keps/prod-readiness/sig-network/1672.yaml

```diff
@@ -0,0 +1,5 @@
+kep-number: 1672
```

Comment:
Can you please split those two into separate PRs? I can easily imagine one being ready for approval and the other not (given that you opened the PR day before deadline...)

Reply:
After reviewing it below - it's even more important. I think 1669 is doable (it's very close to be ready), but 1672 is definitely at high risk.

Reply:
1669 is dependent on 1672 for the terminating condition

Reply:
But that's fine to have 1669 in alpha and 1672 alpha - let's please split them.

Reply:
If you're proposing wekeep 1672 in alpha for v1.22, that's fine with me. But I don't want to split the PR because the context between the two is important.

Reply:
Are you saying we should set the milestone for 1672 to alpha?

Reply:
If I'm being frank, the milestone in this KEP is pretty meaningless to me because the feature gate state will be determined during development based on how the PR review goes. Setting it to beta here just means we have the option to promote it, but that's not really my intention until the development work is done and maturity of the feature is evaluated.

Reply:
I'm adding this to the beta graudation criteria:
```
* Consensus on scalability implications resulting from additional EndpointSlice writes with approval from sig-scalability.
```

Would that be acceptable to you? I agree there are some open questions, but I'm not sure it warrants assuming the feature will be alpha in v1.22. I would like the option to promote this in v1.22 assuming the open concerns are addressed.

Reply:
I think his point is that, should we not get consensus on 1672, the fact that 1669 is in this same PR means 1669 will miss the boat.  If you move 1669 to a new PR, it's ~approved.

Also note that @wojtek-t is in EU, so it's coming on 2am

Reply:
OK, even easier, I will remove the beta milestone now and leave it as alpha, we can discuss in a follow-up if we want beta in v1.22

Reply:
reverted to alpha, PTAL

Reply:
Yeah, I'm just not sure there's time (modulo exception) to discuss :(

Reply:
No problem, my fault for updating my KEPs so last minute ;) 

For what it's worth, I think we addressed most of @wojtek-t 's comments regarding metrics and implications on SLO, so I'm curious what the remaining concerns are.

Reply:
OK - metrics are not strictly needed for alpha - so we have more time to discuss before beta next cycle.

Let me quickly go over it now.